util/awsutil/rds.go

```diff
@@ -11,6 +11,20 @@ import (
 	"github.com/pganalyze/collector/util"
 )
 
+var IdentifierMap *util.TTLMap = util.NewTTLMap(5*60, 30)
+
+func FindRdsIdentifier(config config.ServerConfig, sess *session.Session) (identifier string, err error) {
+	identifier = IdentifierMap.Get(config.AwsDbInstanceID)
+	if identifier == "" {
+		instance, err := FindRdsInstance(config, sess)
+		if err != nil {
```

Comment:
Good idea! I've added that. Though I went with a 10 minute TTL for errors instead of 1 hour because I'm worried that temporary network issues could cause us to drop an hour of logs fairly often.